spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -99,6 +100,7 @@
 
 
 logger = logging.getLogger(__name__)
+UPDATE_DECORATIONS_TIMEOUT = 500  # miliseconds

```

Comment:
As with `UPDATE_TIMEOUT` should this value be configurable in some way? (Maybe a different value is better for different machines)?